static/translations/messages_en.json

```diff
@@ -50,6 +50,7 @@
   "Reply": "Reply",
   "Add a reaction": "Add a reaction",
   "Copy to clipboard": "Copy to clipboard",
+  "Copy Link to clipboard": "Copy Link to clipboard",
```

Comment:
`C`opy, `L`ink are capital, but `t`o and `c`lipboard are small 😅 

Intensional or my mistake? :)

Reply:
I seem to have forgotten to fix this, thanks for the reminder! :)